app/tasks/github_receiver.py

```diff
@@ -110,9 +114,11 @@ def _handle_card(self, board_id, list_id, issue_autocard,
         elif pull_request_autocard and scope == 'pull_request' and \
              action == 'opened':
             self._create_trello_pull_request_card(board_id, list_id, assignee_id)
+        elif merged_pull_request_autocard and merged and action == 'closed':
+            self._create_trello_merged_pull_request_card(board_id, list_id, _assignee_id)
```

Comment:
Should the `_assignee_id` be `assignee_id` instead?

Reply:
Good catch, missed this! Fixed now.